tooling/minimize/tests/ub/tree_borrows/core/offset_access_child_read_disabled.stderr

```diff
@@ -1 +1 @@
-fatal error: UB: Tree Borrows: local read of a pointer with Disabled permission
+fatal error: UB: Tree Borrows: Uniqueness violation: local read after foreign write
```

Comment:
Same here about the "Uniqueness violation:" prefix

Reply:
I kinda liked it that you can classify "what happened" in a way that mostly allows you to not use the names of permissions.

Reply:
So many prefixes don't look great though.
Also, MiniRust is not really about good error messages. This would be something to do in Miri. In MiniRust, referring to the underlying model actually seems like a good thing.